bin/lib/zopen-build

```diff
@@ -1046,6 +1048,10 @@ configure()
 build()
 {
   buildStartTime=$SECONDS
+  if command -V "${ZOPEN_PRE_BUILD_CODE}" >/dev/null 2>&1; then
+    printVerbose "Running ${ZOPEN_PRE_BUILD_CODE}"
+   "${ZOPEN_PRE_BUILD_CODE}" "${ZOPEN_SRC_DIR}"
```

Comment:
can you add a check for the return code being non-zero and exit if so?

Reply:
Done, Mike. Thank you!